### DISCUSSION TITLE
Add support for ICM42688P IMU

### DISCUSSION THREAD

**Discussion by sethml** [qqqlab/madflight#29] [Ideas]
The ICM42688P gyro is in a lot of modern all-in-one flight controllers, and not supported by MadFlight as far as I can see. Porting the driver over from BetaFlight doesn't seem like it'd be too hard. I may do it myself at some point.

**Comment by qqqlab** [comment]
The ICM42688P is also on my wishlist, but didn't find the time yet to work on it. 

I would be great if you implement it and send me a pull request!

**Comment by qqqlab** [comment]
added in v2.0.1
